### PR TITLE
Updates styling & moves tick to the right

### PR DESCRIPTION
**Pull request by andycarrell**
💃

### CODE REVIEW

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 1]
```diff
@@ -8,7 +8,7 @@
 }
 
 .term {
-  width: 8rem;
+  width: 7rem;
```
Comment (andycarrell) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:11]: Don't need to hold space for tick anymore

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss [hunk 2]
```diff
@@ -20,14 +20,9 @@
   text-align: right;
 }
 
-.definition-secondary {
-  flex: 1 1 0%;
-  text-align: right;
-}
```
Comment (andycarrell) [inline app/components/DosesDescriptionList/DosesDescriptionList.module.scss:26]: Just use flex helpers instead

**Review by finnclark — CHANGES_REQUESTED** [review]
Looks great but needs main to be merged in :)

File: app/components/DosesDescriptionList/DosesDescriptionList.module.scss (hunk 2 above)
Comment (finnclark) [reply app/components/DosesDescriptionList/DosesDescriptionList.module.scss:26]: smart

**Comment by andycarrell** [comment]
@finnclark why do I need to merge main??

**Comment by finnclark** [comment]
> @finnclark why do I need to merge main??
> 
> cando tho :D

The dropdown was old

**Review by finnclark — APPROVED** [review]

**Comment by finnclark** [comment]
dis branch is out of date again after #82 got merged

**Pull request merged**